### DISCUSSION TITLE
微调train.log为空，shell显示failed to create process

### DISCUSSION THREAD

**Discussion by donnotuse** [zai-org/ChatGLM3#510] [Q&A]
save_steps 500 --learning_rate 2e-2 --quantization_bit 8 --pre_seq_len 128
+ tee output/advertise_gen_pt-20231203-180834-128-2e-2/train.log
failed to create process.
(chatglm3-demo) PS D:\ChatGLM3\finetune_chatmodel_demo> bash ./scripts/finetune_pt.sh
+ PRE_SEQ_LEN=128
+ LR=2e-2
+ torchrun --standalone --nnodes=1 --nproc_per_node=1 finetune.py --train_format input-output --train_file formatted_data/advertise_gen.jsonl --preprocessing_num_workers 1 --model_name_or_path THUDM/chatglm3-6b --output_dir output/advertise_gen_pt-20231203-180840-128-2e-2 --max_source_length 1024 --max_target_length 128 --per_device_train_batch_size 1 --gradient_accumulation_steps 16 --max_steps 1000 --logging_steps 1 --save_steps 500 --learning_rate 2e-2 --quantization_bit 8 --pre_seq_len 128
+ tee output/advertise_gen_pt-20231203-180840-128-2e-2/train.log
failed to create process.
(chatglm3-demo) PS D:\ChatGLM3\finetune_chatmodel_demo>
这是控制台的信息，我的步骤都是按照文档来的，求助求助

**Comment by donnotuse** [comment]
用的conda，py3.11，torch2.11，cuda11.8

**Reply by zhengheji** [reply]
哥解决了么？我也是这个问题

**Comment by zRzRzRzRzRzRzR** [comment]
#253 在这里讨论
